src/java/htsjdk/samtools/SamReader.java

```diff
@@ -57,6 +57,7 @@ public String toString() {
             }
         }
 
+        public static Type SRA_TYPE = new TypeImpl("SRA", "sra", null);
```

Comment:
Why are you not registering a file extension for SRA index files here? (I take it from the existence of `SRAIndex` above that SRA files can have a companion index file?)